config.example.toml

```diff
@@ -1,6 +1,9 @@
 [app]
     port = 3000
 
+[server]
```

Comment:
rename to `graphql-server` and also add `addr` parameter. default value is 127.0.0.1